build/vsts-ci.yml

```diff
@@ -166,7 +166,7 @@ extends:
             rm -rf /usr/local/bin/2to3
           displayName: MacOS Homebrew bug Workaround
           continueOnError: true
-        - script: brew update && brew install -f --overwrite python@3.12 && brew install libomp && brew link libomp --force
+        - script: brew update && brew install -f --overwrite python@3.13 && brew install libomp && brew link libomp --force
```

Comment:
Do you remember why we do this?  I see it was added in https://github.com/dotnet/machinelearning/commit/59973fc2572532ecbcac35ac59fb8d1164cc4f8f.

Do you also need to fix https://github.com/dotnet/machinelearning/blob/32bac5e395084d1732b18313d2fc2fc74f5b4be6/build/ci/job-template.yml#L74